.travis.yml

```diff
@@ -23,20 +23,9 @@ env:
 
 matrix:
   include:
-    - env: NIX_CHANNEL=https://nixos.org/channels/nixpkgs-unstable
-    - env: NIX_CHANNEL=https://nixos.org/channels/nixos-unstable
-    - env: NIX_CHANNEL=https://nixos.org/channels/nixos-20.03
-
-install:
- - nix --version
- # for cachix we need travis to be a trusted nix user
- - echo "trusted-users = $USER" | sudo tee -a /etc/nix/nix.conf
- - sudo systemctl restart nix-daemon
- - if [ -n "${CACHIX_CACHE}" ]; then travis_retry nix-channel --update; fi
- - if [ -n "${CACHIX_CACHE}" ]; then nix-env -iA cachix -f https://cachix.org/api/v1/install; fi
- - if [ -n "${CACHIX_CACHE}" ]; then cachix use "${CACHIX_CACHE}"; fi
```

Comment:
Do you intentionally no longer install cachix?

Reply:
Yep--I just forgot that I probably need to show my math. :)

I left it out of the initial README so that I could see if domen thought including it made sense (since I'm otherwise tracking what install-nix-action does reasonably closely), but nix-travis-ci installs cachix if `CACHIX_CACHE` is defined (see https://github.com/nix-community/nix-travis-ci/blob/3c33ae750aeaceb3600ca171afb1fe00d485aac3/install.sh#L124-L128).

I have clarified that with him, but I haven't decided where to put it in the README yet.

Reply:
Ideally there would be a commented out environment variable in this file as well, so there it is hard to get it wrong.

Reply:
It's already in the file (but not commented):
https://github.com/nix-community/nur-packages-template/blob/5f885e2333d3a401cc70ad02c56566dd4f62711f/.travis.yml#L16

Reply:
Thanks. I missed that. I guess we just need to comment it out and are done with it?

Reply:
I'll take that as a request :)